platform/broadcom/saibcm-modules/debian/opennsl-modules.init

```diff
@@ -60,14 +68,14 @@ function load_kernel_modules()
 
     modprobe linux-kernel-bde dmasize=$dmasize maxpayload=128 debug=4 dma_debug=1 usemsi=$usemsi
     modprobe linux-user-bde
-    modprobe psample
+    insmod /lib/modules/$(uname -r)/extra/psample.ko
```

Comment:
can you add comments on why you are doing insmod instead of modprobe?

Reply:
Updated